### ISSUE TITLE
How can I put other Sequences inside of Master Sequence

### ISSUE THREAD

**Issue by ricardosferro** [20tab/UnrealEnginePython#609]
Hi Guys,

How can I put other Sequences inside of one Master Sequence inside MovieSceneCinematicShotTrack.

For Example:
I created the Master Sequence and put a MovieSceneCinematicShotTrack, but I can't add others Sequences inside MasterSequence, inside MovieSceneCinematicShotTrack.

Exemplifying better.  There is an image of Unreal "Add Master Sequence" with shots added on Sequencer.  I need build a similar sequence with several shots inside.

to sum up.
The Master Sequence and the shots, I managed to do and work perfectly, but I don't add inside my master sequence.
How can I add sections with another shots created (level sequence) in my Master Sequence?

Thank you.

![image](https://user-images.githubusercontent.com/11772229/48916050-b251ed80-ee78-11e8-95c7-7733b4d3b546.png)

**Comment by 5775486f** [comment]
You can use
shotTrack = sequencer_add_master_track(MovieSceneCinematicShotTrack)# To create a shot track
shotSection = shotTrack.sequencer_track_add_section()#To creat a empty shot
shotSection.SubSequence = your target sequencer

don't forget set section range

**Comment by rdeioris** [comment]
@ricardosferro if you want you can create a markdown page and place it under the docs/example directories with a pull request (or directly into the sequencer api page)
